packages/renderer/src/lib/pod/PodCreateFromContainers.svelte

```diff
@@ -12,7 +12,7 @@ import ContainerIcon from '../images/ContainerIcon.svelte';
 
 let podCreation: PodCreation;
 let createInProgress = false;
-let createError = undefined;
+$: createError = undefined;
```

Comment:
It's not clear to me why it's in reactive form ?

Reply:
old stuff, reverted